.github/workflows/test_publish.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
```

Comment:
We should give these distinctive names so we can find them in the actions pane. Otherwise you end up with a sidebar that looks like this (edx-platform):
<img width="465" alt="image" src="https://user-images.githubusercontent.com/23789/159013499-502702b2-bbd1-4f09-af62-d3e9066b5307.png">

Reply:
Oops copypasta. Good catch.